CMakeLists.txt

```diff
@@ -576,43 +576,76 @@ if (USE_PYTHON)
         if (PYTHON_COPY)
             set(TARGET_PREFIX "${CMAKE_BINARY_DIR}")
             set(PYTHON_EXTRA_MODULES_PATH "${TARGET_PREFIX}/lib/flow123d")
+              
+              # copy python files to build tree 
+              # first get prefix (we support just one prefix)
+              execute_process(
+                  COMMAND python -c "import sys; print sys.prefix"
+                  OUTPUT_VARIABLE PY_PREFIX
+                  OUTPUT_STRIP_TRAILING_WHITESPACE)        
+              execute_process(
+                  COMMAND python -c "import sys; print sys.exec_prefix"
+                  OUTPUT_VARIABLE PY_EXEC_PREFIX
+                  OUTPUT_STRIP_TRAILING_WHITESPACE)        
+              if (NOT (${PY_PREFIX} STREQUAL ${PY_EXEC_PREFIX}) )
+                  message(WARNING "Python exec_prefix (${PY_EXEC_PREFIX}) different then prefix (${PY_PREFIX}). Not supported.")
+              endif()
+                            
+            # get all valid python system paths
+            execute_process(
+              COMMAND python ${CMAKE_SOURCE_DIR}/bin/python/python_path_script.py sys_path --format cmake
+              OUTPUT_VARIABLE PY_PATHS
+              OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-            string(REGEX MATCH "python[^/]*$" PYTHON_SUBDIR ${PYTHON_INCLUDE_DIRS})
-            string(REGEX MATCH "python[0-9]\\.[0-9]" PYTHON_SUBDIR_MINOR ${PYTHON_SUBDIR})
             
-            # copy python files to build tree 
-            # first get prefix (we support just one prefix)
+            # get all subfolders from paths so we can copy them during install
+            # some are located in lib some in lib64 some in local
+            # Python's sys.path contains several paths we must copy in order to assure Python's correct functionality.
+            # We decided to remove part /usr from paths to make them shorter. sys.path:
+            #    /usr/lib64/python2.7
+            #    /usr/lib64/python2.7/site-packages
+            #    /usr/lib/python2.7/site-packages
+            # if we remove prefix we get:
+            #    /lib64/python2.7
+            #    /lib64/python2.7/site-packages
+            #    /lib/python2.7/site-packages
+            # we store first part of each stripped sys.path, remove duplicate and join them with compilation
```

Comment:
It's still unclear what the 'first part' is. You mean: "The script::
1. takes all paths from PYTHON_PATH
2. for each extract the name of the first subdirectory under prefix
3. make list unique 

so we get something like:
"./lib;./lib64;./local"

If this is the function of the script, the name of the output variable should be rather PY_PREFIX_SUBDIR, since PY_SUB_PATHS  evokes something deeper then PY_PATHS, i.e. quite opposite thing.